src/layer/vector/Polyline.js

```diff
@@ -172,6 +172,21 @@ export var Polyline = Path.extend({
 		return this.redraw();
 	},
 
+	// @method setLastLatLng(latlng: LatLng, latlngs? LatLng[]): this
+	// Modifies the last point of the polyline to be the specified point.
+	// By default, adds to the first ring of the polyline in case of a
+	// multi-polyline, but can be overridden by passing a specific ring as a
+	// LatLng array (that you can earlier access with
+	// [`getLatLngs`](#polyline-getlatlngs)).
+	setLastLatLng: function (latlng, latlngs) {
+		latlngs = latlngs || this._defaultShape();
```

Comment:
There is no guarantee that `latlngs` is related at all to the polyline's coordinate set. Not only for the obvious case of specifying an explicitly unrelated `latlngs`, but also for the subtle cases where shallow copies of the arrays are created. In those cases, this method just modifies the parameter passed by reference, not the data contained in `this`.